release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1669,6 +1669,147 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, the `oc adm drain --delete-local-data` command was not supported in the 4.18 `oc` cli tool. With this release, the command has been updated to `oc adm drain --delete-emptydir-data`. (link:https://issues.redhat.com/browse/OCPBUGS-44318[*OCPBUGS-44318*])
+
+* Previously, US East (`wdc04`), US South (`dal13`), Sydney (`syd05`), and Toronto (`tor01`) regions were not supported for {ibm-power-server-title}. With this release, these regions, which include `PowerEdgeRouter` (PER) capabilities, are supported for {ibm-power-server-title}.
+(link:https://issues.redhat.com/browse/OCPBUGS-44312[*OCPBUGS-44312*])
+
+* Previously, during a {gcp-first} installation, when the installation program was creating filters with large numbers of returned data, for example for subnets, it exceeded the quota for the maximum number times that a resource can be filtered in a specific period. With this release, all relevant filtering is moved to the client so that the filter quotas are not exceeded and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44193[*OCPBUGS-44193*]) 
+
+* Previously, during an {aws-first} installation, the installer validated all the tags in the `install-config.yaml` file only when you set `propogateTags` to true. With this release, the installer validates all the tags in the `install-config.yaml` file.
+(link:https://issues.redhat.com/browse/OCPBUGS-44171[*OCPBUGS-44171*])
+
+* Previously, if the `RendezvousIP` matched a substring in the `next-hop-address` field of a compute node configuration, it reported a validation error. The `RendezvousIP` must match a control plane host address only. With this release, a substring comparison for `RendezvousIP` is used against a control plane host address only, so that the error no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-44167[*OCPBUGS-44167*])
+	
+* Previously, when you deleted a cluster in {ibm-power-server-title}, the Transit Gateway connections were cleaned up. With this release, if the `tgName` is set, {rh-openstack-first} will not clean up the Transit Gateway connection when you delete a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-44162[*OCPBUGS-44162*])
+
+* Previously, when installing a cluster on an {ibm-title} platform and adding an existing VPC to the cluster, the {cap-ibm-first} would not add ports 443, 5000, and 6443 to the security group of the VPC. This situation prevented the VPC from being added to the cluster. With this release, a fix ensures that the {cap-ibm-first} adds the ports to the security group of the VPC so that the VPC gets added to your cluster.(link:https://issues.redhat.com/browse/OCPBUGS-44068[*OCPBUGS-44068*])
```

Comment:
```suggestion
* Previously, when installing a cluster on an {ibm-title} platform and adding an existing VPC to the cluster, the {cap-ibm-first} would not add ports 443, 5000, and 6443 to the security group of the VPC. This situation prevented the VPC from being added to the cluster. With this release, a fix ensures that the {cap-ibm-first} adds the ports to the security group of the VPC so that the VPC gets added to your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-44068[*OCPBUGS-44068*])
```